### ISSUE TITLE
Is there a way to get ACK when the capture starts

### ISSUE THREAD

**Issue by maltekliemann** [saleae/SaleaeSocketApi#15]
**TL;DR** Is there a way to start a capture and get an acknowledgment for when the capture _starts_?

The problem, roughly, is the following: If `capture` is called and a signal arrives at the recording device, say, a nanosecond later, will that signal be recorded? Probably not. What about a second later? Probably. I would like to know for sure, in other words, I would like to get an acknowledgment which, one received by the host, guarantees that the recording is running. I'm fine with waiting a couple of seconds.

But it appears to me as if there is no way to know when exactly a capture starts. For example, from the documentation:

> Socket Command: capture
>
> This command starts a capture. It will return `NAK` if an error occurs.
>
> The function will return `ACK` when the capture is complete.
